### PR TITLE
Rework discount dan's sauce

### PR DESCRIPTION
**Pull request by yackemflam**
It now deals damage at as a random type!

### CODE REVIEW

**Comment by ghost** [comment]
holy mother of shitcode

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -3577,7 +3577,17 @@
...
+						if(1)
+							H.adjustToxLoss(0.5)
+						if(2)
+							H.adjustToxLoss(0.5)
+						if(3)
+							H.adjustToxLoss(0.5)
+						if(4)
+							H.adjustToxLoss(0.5)
+						if(5)
+							H.adjustToxLoss(0.5)
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm:3590]: You sure about that random type thing?

Also holy shit you quintupled the damage

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -3588,7 +3598,20 @@
...
+							H.adjustBruteLoss(1)
+						if(3)
+							H.adjustFireLoss(1)
+						if(4)
+							H.adjustOxyLoss(1)
+						if(5)
+							H.adjustToxLoss(1)
+							H.adjustOxyLoss(1)
+							H.adjustFireLoss(1)
+							H.adjustBruteLoss(1)
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm:3614]: Jesus

**Comment by Kammerjunk** [comment]
memes

**Comment by kilozombie** [comment]
What _is_ coding anyway

**Comment by ghost** [comment]
how can coding be real if our brains aren't real

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 3]
```diff
@@ -3577,7 +3577,17 @@
 				if(prob(10))
 					to_chat(H,"<span class='warning'>You really don't feel very good.</span>")
 				if(prob(5))
-					H.adjustToxLoss(0.1)
+					switch(pick(1,2,3,4,5))
```
Comment (ghost) [inline code/modules/reagents/Chemistry-Reagents.dm:3580]: Why this?

Comment (Exxion) [reply code/modules/reagents/Chemistry-Reagents.dm:3580]: It's a meme PR anyway

At least I hope it is

Comment (ghost) [reply code/modules/reagents/Chemistry-Reagents.dm:3580]: I'm.. pretty sure this isn't a meme PR.
Just look at his PR history.

Comment (Exxion) [reply code/modules/reagents/Chemistry-Reagents.dm:3580]: I noted in one place that he quintupled the damage, but I didn't point out that in another, he nearly *octupled* it, with a 1/5 chance of quadrupling *that*

I'm treating it as a meme PR, at least, unless he says something to justify it

Comment (yackemflam) [reply code/modules/reagents/Chemistry-Reagents.dm:3580]: It's half memey.
Discount Dans food is hardly lethal. Even in large quantity (200+ units of dans sauce.)
If something is so slow that it's damage has to to be quintupled to show any real effect, it's hardly there.
(also you can heal off the damage easy)

Comment (Shadowmech88) [reply code/modules/reagents/Chemistry-Reagents.dm:3580]: Discount Dans food isn't supposed to be lethal or even somewhat damaging.
It's supposed to be an annoyance.

Comment (Probe1) [reply code/modules/reagents/Chemistry-Reagents.dm:3580]: You have reached the first stage of coder awareness @yackemflam 

1.  I CAN code this.

You have to grow to the next stage.

2.  I SHOULD code this.

**Pull request closed without merging**